planning/autoware_behavior_path_avoidance_by_lane_change_module/src/scene.cpp

```diff
@@ -32,9 +32,12 @@
 #include <optional>
```

Comment:
```suggestion
#include <algorithm>
#include <optional>
```

To pass https://results.pre-commit.ci/run/github/429651616/1715682762.0qguD79LT5yQcQVUZwVcpQ